Bundle/CoreBundle/Entity/View.php

```diff
@@ -554,22 +553,28 @@ public function removeWidgetMap(WidgetMap $widgetMap)
     }
 
     /**
-     * Get widgets ids as array.
+     * Get WidgetMaps for View and Templates.
      *
-     * @return array
+     * @return WidgetMap[]
      */
-    public function getWidgetsIds()
+    public function getWidgetMapsForViewAndTemplates($viewContext = null)
```

Comment:
Furthermore, what is its role ? Can you add a PHP_doc statement for it ?
Is this variable really optional ?